assets/src/edit-story/components/inspector/document/publish/index.js

```diff
@@ -157,20 +178,19 @@ function PublishPanel() {
   );
 
   const authorLabel = __('Author', 'web-stories');
-  const use12HourFormat = is12Hour(timeFormat);
   return (
     <SimplePanel name="publishing" title={__('Publishing', 'web-stories')}>
       <Row>
         <FieldLabel>{__('Publish', 'web_stories')}</FieldLabel>
         <StyledButton
-          aria-pressed={showDatePicker}
+          aria-pressed={hasSidebar}
           aria-haspopup={true}
-          aria-expanded={showDatePicker}
+          aria-expanded={hasSidebar}
```

Comment:
Expanded? It's not really expanded, is it?

Also, the icon doesn't actually change to an up arrow, so maybe the arrow icon should be completely removed now?
![Screen Shot 2020-04-01 at 2 02 41 PM](https://user-images.githubusercontent.com/637548/78171231-2a8fd200-7422-11ea-905f-68d55291d32c.png)

Reply:
Maybe change the arrow to a calendar icon, if we have one such? Tagging in @samitron7 here.